flexeval/core/language_model/litellm_api.py

```diff
@@ -25,6 +25,8 @@ class LiteLLMChatAPI(OpenAIChatAPI):
         developer_message: Instructions to the model that are prioritized ahead of user messages.
             Previously called the system prompt.
         string_processors: A single or a list of StringProcessor objects to process the model's output.
+        ignore_seed: If True, ignore the seed specified in default_gen_kwargs.
```

Comment:
How about let the class ignore seed in `gen_kwargs` in general?
It just prevents `seed` being passed to Anthropic in broader contexts, not only through `default_gen_kwargs`.

Not sure when it will be helpful, but feels like it is more intuitive behavior given the option name `ignore_seed`.